_init.py

```diff
@@ -7,7 +7,9 @@
 except ModuleNotFoundError:
     # Stub out this call so that we can run the external_libraries script
     # without needing the entire codebase.
-    def get_config_provider(*args, **kwargs):
+    def get_config_provider(
```

Comment:
We can remove `config_provider` from _init and move it to `app`, it'll eliminate this import.